sympy/integrals/prde.py

```diff
@@ -979,6 +992,9 @@ def pad(rep):
         # 'Kmat' is the same of 'Qmat'
         Qmat = Matrix(Kmat)
 
+    V = Qmat.rowspace()
```

Comment:
Okay, so should it be like `F` itself is given changed already as input (i.e `fa/fd` is already appending to `F` before input to `linear_relations_in_Q`) and then after that from the all the nullspace matrices we obtain the one put the coresponding `ri` of `fa/fd` to `1`?